### ISSUE TITLE
Product quantity should factor in ingreedy-py 'weight' field when present

### ISSUE THREAD

**Issue by jayaddison** [openculinary/ingredient-parser#15]
The [ingreedy-py](https://github.com/openculinary/ingreedy-py) parser results may include both a product `quantity` and also a product `weight` in cases where an ingredient line specifies both.

For example:

```
>>> from ingreedypy import Ingreedy
>>> Ingreedy().parse('4 (28 ounce) cans baked beans, drained')
{'unit': 'ounce', 'unit_type': 'english', 'amount': 4, 'ingredient': 'cans baked beans, drained', 'weight': 28}
```

However, as can be seen in the [recipe](https://www.reciperadar.com/#search&action=view&id=AABQjgpohkzqk57v1KwvaqKFzzTz85TENJ) shown below, the `ingredient-parser` currently only calculates the product quantity based on the `amount` field:

![image](https://user-images.githubusercontent.com/55152140/71265620-7d93bc80-233e-11ea-96ff-990f0b2d96c3.png)

Rendered: 4oz ~= 115g
Expected: 4 x 28oz ~= 3.15kg

**Comment by jayaddison** [comment]
This turned out to be a little more complicated than initially anticipated!

As part of https://github.com/openculinary/ingreedy-py/pull/8, the `ingreedypy` library has now been updated to return a list of quantities for each ingredient line, and also to return quantities which factor in both the multiplier and amount of an individual product.

To explain that slightly differently: the `ingreedypy` library will now parse `two (28 ounce) can tomatoes` into a single quantity of `56 ounces`.  This removes the need for any multiplication of weights and quantities within the `ingredient-parser` codebase.

**Comment by jayaddison** [comment]
Resolved - quantity is now displayed as `3.18kg`, which is correct.

![image](https://user-images.githubusercontent.com/55152140/71782855-bbe3d980-2fd6-11ea-866b-8d135f3ddfe7.png)
